src/utils/validatePhoneNumbers.tsx

```diff
@@ -32,12 +32,3 @@ export const countryCodeValidator = (code: string): boolean => {
   );
   return code[0] === "+" && countryCodesList.includes(code.substring(1));
 };
-
-export const validatePhoneNumbers = (phoneNumbers: string[]): boolean => {
```

Comment:
Hmmm is this being used by the OTP flow?

Reply:
Not this function. OTP flow uses other functions in this file. I created this for the checkout previously~